src/runtime_src/core/pcie/windows/system_windows.cpp

```diff
@@ -19,7 +19,7 @@
 // file from importing symbols from libxrt_core we define this source
 // file to instead export with same macro as used in libxrt_core.
 #define XCL_DRIVER_DLL_EXPORT
-
+#define XRT_CORE_PCIE_WINDOWS_SOURCE
```

Comment:
Could you remind me the reason we need to define this value in the source code as opposed to having CMake define it.